tests/Behat/MinkContext.php

```diff
@@ -16,6 +17,23 @@ class MinkContext extends DrupalExtensionMinkContext {
 
   use UtilityTrait;
 
+  /**
+   * Mapping between human readable element labels and CSS selectors.
+   *
+   * @var array
+   */
+  private $breadcrumbElements = [];
+
+  /**
+   * TransformationContext constructor.
+   *
+   * @param array $breadcrumb_elements
+   *   Breadcrumb elements mapping.
+   */
+  public function __construct(array $breadcrumb_elements = []) {
```

Comment:
There's no need to have these data coming from the context. Let's remove all of that and have two steps:
`@Then the breadcrumb (links|trail) should be :links`
`@Then the breadcrumb active element should be :text`
Below I'll make an example for the scenario.